client-mobile/src/components/organisms/UpdateGroceryItemsForm/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { IconName } from "@client/components/atoms/Icon/@types"
+import { AddIngredientForm } from "@client/components/molecules/AddIngredientForm"
+import { Banner } from "@client/components/molecules/Banner"
+import { Button } from "@client/components/molecules/Button"
+import { ButtonSize } from "@client/components/molecules/Button/@types"
+import { MultiSelectField } from "@client/components/molecules/MultiSelectField"
+import { IUpdateGroceryItemsFormProps } from "@client/components/organisms/UpdateGroceryItemsForm/@types"
+import { useUpdateGroceryItemsFormState } from "@client/components/organisms/UpdateGroceryItemsForm/hooks/useUpdateGroceryItemsFormState"
+import { addGroceryItemsFormDefaultStyles } from "@client/components/organisms/UpdateGroceryItemsForm/styles"
+import { ScrollView, View } from "dripsy"
+
+export const UpdateGroceryItemsForm: React.FC<IUpdateGroceryItemsFormProps> = ({
+  ingredients,
+  isUpdatingGroceryItems,
+  isAddingIngredient,
+  onUpdateGroceryItems,
+  onAddIngredient,
+  shoppingList,
+}) => {
+  const {
+    areIngredientsInShoppingList,
+    handleSubmit,
+    handleUpdateGroceryItems,
+    control,
+    unshiftIngredientInFormIngredients,
+    watchedIngredients,
+  } = useUpdateGroceryItemsFormState({
+    ingredients,
+    onUpdateGroceryItems,
+    shoppingList,
+  })
+
+  const { button, childrenWrapper, outerWrapper } = addGroceryItemsFormDefaultStyles
+
+  const getButtonText = () => {
```

Comment:
```suggestion
  const getButtonText = (): keyof ITranslation => {
```

Also, please move this function to `useUpdateGroceryItemsFormState`.